arrayMethodstask/script.js

```diff
@@ -0,0 +1,14 @@
+let matrix = [
+[-2, 0, 9, 5, 8],
+[1, 18, -59, 3, 3, 6, 4],
+[101, 24, -79],
+[7, 7]
+]
+const reduceSingleItemFun=(arr)=>arr.reduce((prev,curr)=> prev+ curr ,0)
```

Comment:
When you're writing code, you always want to aim for code readability in addition to code correctness.

`const reduceSingleItemFun = (arr) => arr.reduce((prev, curr) => prev + curr, 0)`